shared/src/main/scala/pl/metastack/metatime/Time.scala

```diff
@@ -14,11 +15,17 @@ trait Component extends Ordered[Component] {
 
   def to(until: Component): Range = ???
 
-  override def compare(that: Component): Int = ???
+  def unix(): Unix
+
+  override def compare(that: Component): Int =
+    this.unix().value compare that.unix().value
```

Comment:
Let's stick to the convention of using this notation only for operators. Use `.compare()` instead.